File: cpp/ccc11j1.cpp
// Which Alien (CCC1 L1).cpp : Defines the entry point for the console application.

#include <iostream>

using namespace std;

int main()
{
	int antennae, eyes;
	cin >> antennae;
	cin >> eyes;

	if (antennae >= 3 && eyes <= 4)
		cout << "TroyMartian" << endl;
	if (antennae <= 6 && eyes >= 2)
		cout << "VladSaturnian" << endl;
	if (antennae <= 2 && eyes <= 3) {
		cout << "GraemeMercurian" << endl;
	}

    return 0;
}

File: java/ccc11j1.java
import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;

public class ccc11j1 {
	
	static Scanner sc = new Scanner(System.in);
	static List<String> pss = new ArrayList<String>();

	public static void main(String[] args) {
		int ants = sc.nextInt();
		int eyes = sc.nextInt();
		
		if(ants >= 3 && eyes <= 4)
			pss.add("TroyMartian");
		if(ants <= 6 && eyes >= 2)
			pss.add("VladSaturnian");
		if(ants <= 2 && eyes <= 3)
			pss.add("GraemeMercurian");
		
		if(!pss.isEmpty()){
			for(String s : pss)
				System.out.println(s);
		}
	}

}
